content/docs/command-reference/fetch.md

```diff
@@ -1,7 +1,7 @@
 # fetch
 
-Get tracked files or directories from
-[remote storage](/doc/command-reference/remote) into the <abbr>cache</abbr>.
+Download <abbr>cached</abbr> files or directories from
```

Comment:
hmm .. not sure I like it this way .. previous version was easy enough. Why do we change it this way.

I'm fine to change Get to Download ... cached files - we usually use `tracked` everywhere else

Reply:
> cached files - we usually use tracked everywhere else

But we usually refer to files in the workspace so that wouldn't apply to fetch. I'm just trying to state what this command does in the simplest and most direct way: it downloads files to the cache.

Reply:
Changed it to

> Download files or directories from
> [remote storage](/doc/command-reference/remote) to the <abbr>cached</abbr>.

without saying "tracked", to avoid confusion.